### DISCUSSION TITLE
Analysis 26a, 26b, 26c, 26d, 26b-cice6, and 27

### DISCUSSION THREAD

**Discussion by cecilehannay** [NCAR/amwg_dev#183] [TF5: Coupled SE/MOM6]
Please to add any plot helping with the analysis of Analysis 26a, 26b, 26b-cice6, and 27 to the CAM7 meeting slide deck: https://docs.google.com/presentation/d/1AQfO3dCa41bOT7-2aWjdwY4MI8xCCGOw/edit?usp=sharing&ouid=110527899490311587217&rtpof=true&sd=true

**Comment by JulioTBacmeister** [comment]
25b has gone to Year 97.  Lab Sea ice fraction remains low.  
![image](https://user-images.githubusercontent.com/9221710/194090733-be5d2cc5-3b07-4031-9118-5a8e92df7205.png)

**Comment by cecilehannay** [comment]
Thanks @JulioTBacmeister. 
Cheyenne was slow on the slow side yesterday, so we didn't get much more years.  
When 26c and 26d are long enough, it would be nice to get the ocean plots too. (@gustavo-marques,@fobryan3, @klindsay28).

**Comment by klindsay28** [comment]
26d is getting a large negative frozen runoff into the Lab Sea. The magnitude is comparable to the positive frozen runoff in other cases. This seems problematic to me.
![image](https://user-images.githubusercontent.com/15570875/194122900-f346d41f-fa46-473b-837b-15a24e5820b4.png)

**Comment by JulioTBacmeister** [comment]
What does negative frozen runoff mean?  The land model is acquiring sea-ice and turning it into snow?  Can this happen if sea level drops and sea-ice is "beached"?

**Comment by fobryan3** [comment]
I think if the land type is glacier and all the snow melts they create
negative frozen runoff to keep adding snow and keep it snow covered.

On Wed, Oct 5, 2022 at 11:31 AM JulioTBacmeister ***@***.***>
wrote:

> What does negative frozen runoff mean? The land model is acquiring sea-ice
> and turning it into snow? Can this happen if sea level drops and sea-ice is
> "beached"?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NCAR/amwg_dev/discussions/183#discussioncomment-3807603>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALNOABIWQSPU3RR2HC6FRSLWBW3QXANCNFSM6AAAAAAQZFDERQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
_______________________________________________________________________
Dr. Frank O. Bryan
Oceanography Section , Climate and Global Dynamics Laboratory
National Center for Atmospheric Research, P.O. Box 3000, Boulder, CO 80307
USA
e-mail: ***@***.*** / voice : (303) 497-1394  /   fax   : (303)
497-1700

**Reply by adamrher** [reply]
I don't follow this at all. there is no requirement that glacier land units be snow covered. @billsacks does this sound familiar?

**Reply by billsacks** [reply]
Yes, running ice sheets without a dynamic ice sheet model has always created problematic situations. The solutions we have in place tend to work okay on net if you are near equilibrium, but in transient situations can create problematic fluxes in order to try to maintain water balance in a fundamentally out-of-balance situation that doesn't have explicit tracking of ice sheet mass.

@fobryan3 is effectively correct, although some references to snow there should actually be ice: There is no requirement that glacier land units be snow covered, but once the snow melts, then the bare ice can start melting. If we're not running with a dynamic ice sheet model (CISM), then we have a requirement that the ice volume stay fixed in time, so we suck "ice" back out of the river / ocean to maintain this fixed ice volume. For more details and rationale, see https://escomp.github.io/ctsm-docs/versions/master/html/tech_note/Glacier/CLM50_Tech_Note_Glacier.html#computation-of-the-surface-mass-balance – especially starting with the paragraph that begins, "If glc_dyn_runoff_routing is false".

**Reply by adamrher** [reply]
I see. Thanks, Bill. Looking at the document, it states that this only occurs for Greenland and Antarctica (is it all ice surfaces of the _mec type only?).

If we are concerned about unrealistic freshwater fluxes to the ocean, then why not turn on CISM over Greenland? Then the ice melt would be routed into the ocean, and the amount of land ice would not be fixed globally (causing unrealistic corrections).

**Reply by billsacks** [reply]
> Looking at the document, it states that this only occurs for Greenland and Antarctica (is it all ice surfaces of the _mec type only?).

Yes, that's right. This is based on the GLACIER_REGION map on the surface dataset and the associated glacier region behavior namelist items. We tried to strike a compromise here: this behavior is needed currently to get reasonable surface mass balance calculations, but we understand that it can cause problematic fluxes, so the compromise was to enable it over the ice sheets but not over other glaciers.

> If we are concerned about unrealistic freshwater fluxes to the ocean, then why not turn on CISM over Greenland?

I think that, when this has come up in the past, the main scientific problem with turning on CISM by default has been the challenges of spinning up CISM to a near-equilibrium state, and concerns that if you start it off far from equilibrium, you could do more harm than good. But this is really a question that should be directed to Bill Lipscomb or others in the LIWG.

**Reply by adamrher** [reply]
Ah OK, thanks for the reminder about the GLACIER_REGION's. I don't know that we need the most realistic ice sheet at this stage, we are just trying to get cam-se and mom6 to play nice with each-other in this phase. The JG/BG greenland ice sheet is fairly spun-up so we could start with that ice sheet to get something ~equilibrated. Let me huddle with the group and possibly reach out to LIWG.

**Comment by JulioTBacmeister** [comment]
Excuse a naive question. Why is negative runoff to maintain constant glacier thickness a bigger problem than regenerating the zeroed-out snow pack in the Arctic and Antarctic via snowfall?

**Reply by adamrher** [reply]
i think it's b/c melting rates are unbounded and can be an order of magnitude larger than snow accumulation.

**Reply by billsacks** [reply]
Note, though, that that source of negative ice runoff is balanced by positive liquid runoff. An issue could be that they arrive in slightly different places, but other than that, the net effect should just be a heat flux.

**Reply by adamrher** [reply]
This gives me a lot to chew on. This is a local issue; globally, the negative fluxes exactly offset the ice melt contribution to runoff. But as you say there are likely issues locally because the positive runoff due to ice melt and the negative runoff correcting for this ice melt, occur in different places.

**Reply by billsacks** [reply]
And to clarify: the negative and positive runoff terms should arrive in roughly the same region; I'm just not sure they arrive in exactly the same place due to differences in how the river model routes positive liquid runoff, negative liquid runoff, and ice runoff, and how these then get mapped to the ocean grid. (I can't remember the details of each of these different routings and mappings, and they have changed over time.)

**Reply by adamrher** [reply]
@klindsay28 is the lab sea liquid runoff much larger in 26d, which should be offsetting some of large negative frozen runoff?

**Reply by klindsay28** [reply]
For the Lab Sea, it doesn't look like lrunoff has gone up to compensate for decreased frunoff. It looks like frunoff in 26d is less than or comparable to what it is in 26c.
![image](https://user-images.githubusercontent.com/15570875/194178207-af89acc1-b31c-47f7-bcdb-535f86db547b.png)
![image](https://user-images.githubusercontent.com/15570875/194178218-91ff00aa-b2f2-4228-a338-45ba3f8eb766.png)
I also have the same plots for the Southern Hemisphere (SH). The change in SH frunoff is about -1e8 kg s-1. It doesn't look like SH lrunoff increased by 1e8 kg s-1.
![image](https://user-images.githubusercontent.com/15570875/194178622-0dae9293-d48a-49c0-b608-9e735c378f35.png)
![image](https://user-images.githubusercontent.com/15570875/194178636-ab221df7-8af8-4803-b4b1-a6040698d09b.png)

**Reply by adamrher** [reply]
I'm not going to say the negative frozen LabSea fluxes aren't a problem, but (1) it's about about a 0.5E7 kg/s reduction in frozen fluxes, which only represents a couple tick marks in the liquid runoff (so by mass, the changes in frozen runoff is much less than the total water runoff), and (2) these negative fluxes should go away as snow accumulates over Greenland, and the slope of the frunoff plot indicates these may decay to less problematic values in ~decades.

I say we keep running 26c, 26d.

**Comment by cecilehannay** [comment]
The slide deck for Oct 7 is [here](https://docs.google.com/presentation/d/1toIsml5d9Ibpm2ml0qouB5G5wOVXuSW-/edit?usp=sharing&ouid=110527899490311587217&rtpof=true&sd=true)

Please feel to include your analysis and send a note to cecile if you need access permission.

**Comment by cecilehannay** [comment]
It looks like 26d didn't recover:

![197600917-13922bdc-9a53-4eee-92e3-5b9ee3f0db8d](https://user-images.githubusercontent.com/9723220/197601722-51bbf01e-4dcd-4e18-88c5-bdb408b261b5.png)

**Reply by JulioTBacmeister** [reply]
It'll be interesting to track this down ... I tend to agree with what was being said at Friday's meeting.  Maybe 26c was just "lucky".  Is there any interest in OS in running a perturbed version of 26c?

**Comment by cecilehannay** [comment]
Slides for co-chair meeting Oct 25 are [here](https://docs.google.com/presentation/d/1oQV2S0lciEqtRgkoHzcJ0t8WMlhL3Pos/edit?usp=sharing&ouid=110527899490311587217&rtpof=true&sd=true)
Please feel free to update.

**Comment by fobryan3** [comment]
I would be interested in seeing a pertlim of 26c. If we have time to burn
in the next 6 days that is a good candidate.

26d did have comparatively very large negative frozen runoff early in the
run which set it apart from other cases, so it may not be totally "lucky"
that 26c is stable.

On Mon, Oct 24, 2022 at 1:40 PM JulioTBacmeister ***@***.***>
wrote:

> It'll be interesting to track this down ... I tend to agree with what was
> being said at Friday's meeting. Maybe 26c was just "lucky". Is there any
> interest in OS in running a perturbed version of 26c?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NCAR/amwg_dev/discussions/183#discussioncomment-3954367>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALNOABOOGYM67QSBZSMSRQLWE3Q33ANCNFSM6AAAAAAQZFDERQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
_______________________________________________________________________
Dr. Frank O. Bryan
Oceanography Section , Climate and Global Dynamics Laboratory
National Center for Atmospheric Research, P.O. Box 3000, Boulder, CO 80307
USA
e-mail: ***@***.*** / voice : (303) 497-1394  /   fax   : (303)
497-1700
